modular_skyrat/master_files/code/modules/mob/living/carbon/human.dm

```diff
@@ -0,0 +1,13 @@
+/mob/living/carbon/human/ZImpactDamage(turf/T, levels)
+	//Non cat-people smash into the ground
+	if(!isfelinid(src) && !istajaran(src))
+		return ..()
+	//Check to make sure legs are working
+	var/obj/item/bodypart/left_leg = get_bodypart(BODY_ZONE_L_LEG)
+	var/obj/item/bodypart/right_leg = get_bodypart(BODY_ZONE_R_LEG)
+	if(!left_leg || !right_leg || left_leg.bodypart_disabled || right_leg.bodypart_disabled)
+		return ..()
+	Paralyze(1 SECONDS)
```

Comment:
I guess this might make it less tedious. We can always tweak this at a later time.
```suggestion
	Paralyze(0.5 SECONDS)
```